app/views/chugs/_form.html.erb

```diff
@@ -10,7 +10,7 @@
       Tijd (in seconden)
     </div>
     <div class="w-full">
-      <%= f.text_field :time, step: :any, inputmode: 'decimal' ,class: input_classes, required: true, placeholder: "_.__", value: '',
+      <%= f.text_field :time, step: :any, inputmode: 'decimal' ,class: input_classes, required: true, placeholder: "_.__", value: @chug.time != 0.0 ? @chug.time : '', autofocus: true,
```

Comment:
Volgens mij doet @chug&.time het hier hetzelfde en scheelt de ternary

Reply:
Ah dat is dus ook waarom het een text field is, misschien kunnen we hetzelfde alsnog krijgen met een number field zodat een telefoon alleen het cijfertoetsenbord laat zien 😅